### DISCUSSION TITLE
adstock transformation w.r.t. to channel nature

### DISCUSSION THREAD

**Discussion by Emnlv** [facebookexperimental/Robyn#765] [Q&A]
Are there some literature review, or do you have suggestion to better understand pre-setting hyperparameters for geometric, weibull_cdf and weibull_pdf for every specific channel?

For example: a digital channel can behave different from a non-digital one.
In additon, are we sure that is correct to apply for every channel the same kind of transformation? i.e., for a nature perspective, can be that a specific channel has more relation with a geometric adstock transformation, whilst another one can be better "synthesized" by a weibul cdf or pdf?

Which setting suggestions can you give me for weibull_pdf about those channel?
Here the boundaries for every channel.
 _alphas = c(>0, <inf),
 _gammas = c(>0, <=1),
 _scales = c(0, <inf),
 _shapes = c(>=0, <=1),

TV
Social
Display
Video Platform
Print
Radio
Addressable
Out of Home

Thanks in advance

**Comment by brettackers** [comment]
Hi @Emnlv did you get to the bottom of this? We're facing the same dillema at the moment
